awd/crossref.py

```diff
@@ -46,7 +46,7 @@ def get_metadata_extract_from(work_record: dict[str, Any]) -> dict[str, Any]:
     ]
     work = work_record["message"]
     value_dict: dict[str, Any] = {}
-    for key in [k for k in work.keys() if k in keys_for_dspace]:
```

Comment:
Is `work` a dictionary or a list of strings? 🤔

Reply:
`dict`, see `tests/fixtures/crossref_work_record.json`